bench/suite/src/main/scala/org/scalameta/bench/SymbolParser.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameta.bench
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Measurement, Mode, Fork}
+import org.langmeta.semanticdb.Symbol
+import org.langmeta.semanticdb.Signature
+
+object SymbolParser {
+  val symbol: String = Symbol.Global(
```

Comment:
You can use the parser to create a symbol from a string `Symbol("_root_.scala.Predef.Int#")`